src/shogun/base/AnyParameter.h

```diff
@@ -39,12 +39,17 @@ namespace shogun
 		// has automatically computed value
 		AUTO = 1u << 10,
 
+		// is const
 		READONLY = 1u << 11,
 
-		// an executable function
-		RUNFUNCTION = 1u << 12,
+		// a class member function with side effects
```

Comment:
do we want to allow this even? I'd prefer if we didnt ... landslide risk

Reply:
This is used by some classes, like RandomMixin to set the random seed (`set_random_seed` is not const). The safety net we have here is that you can only call these functions using `run`, rather than `get`.